### PR TITLE
small tidy up

### PR DESCRIPTION
**Pull request by pouvik**
While this is not massive it looks better and is correct etiquette to have the correct code.  This also creates a unified approach to br's

Signed-off-by: Deaks <bryandeakin30@gmail.com.com>

### CODE REVIEW

**Comment by Oldiesmann** [comment]
We don't use the ` />` with img or hr so why with br? The only place I'm finding `<br />` where it's unnecessary is a single line in Sources/ManageErrors.php.

File: SSI.php [hunk 1]
```diff
@@ -523,7 +523,7 @@ function ssi_queryPosts($query_where = '', $query_where_params = array(), $query
 		censorText($row['subject']);
 		censorText($row['body']);
 
-		$preview = strip_tags(strtr($row['body'], array('<br>' => '&#10;')));
+		$preview = strip_tags(strtr($row['body'], array('<br />' => '&#10;')));
```
Comment (albertlast) [inline SSI.php:526]: html5 define ``<br>`` as valid
and ``<br/>`` is wrong.

**Review by albertlast — CHANGES_REQUESTED** [review]

**Comment by pouvik** [comment]
to be precise and I can give multiple examples of this `<br />` is as accepted as much as `<br>` in html 5

The idea behind the PR is to have a uniformed approach so all br tags are the same, as it is mixed up quite abit depending where you look.  I am more than happy to change  existing br / to b and check for the other tags to make the consistent. However having both br / and br is not good practice.

**Comment by albertlast** [comment]
why then not the other way around?
and change all ``<br />`` to ``<br>``

**Comment by pouvik** [comment]
i am doing it now also going to checdk for others while im at it :)

**Pull request closed without merging**